waspc/data/Generator/templates/sdk/wasp/auth/useAuth.ts

```
@@ -25,14 +25,10 @@ function createUserGetter(): Query<void, AuthUser | null> {
       const userData = superjsonDeserialize<AuthUserData | null>(response.data)
       return makeAuthUserIfPossible(userData)
     } catch (error) {
-      if (error.response?.status === 401) {
-        return null
-      } else {
-        throw handleApiError(error)
-      }
```

If we get a 401 now, it is truly unexpected, so we let it throw.